lib/kitchen/element_base.rb

```diff
@@ -460,11 +466,11 @@ def copy(to: nil)
     def paste
       # See `clone` method for a note about namespaces
       block_error_if(block_given?)
-
       temp_copy = clone
       temp_copy.raw.traverse do |node|
         next if node.text? || node.document?
 
+        document.record_id_paste(node[:id])
         node[:id] = document.modified_id_to_paste(node[:id]) unless node[:id].blank?
```

Comment:
I know that `record_id_paste` is doing a check for blank IDs, but it seems a little weird here to a some other developer reading this to see a blank check on line 474 but not on 473.  Can we wrap both lines in one check?

```ruby
if node[:id].present?
  document.record_id_paste(node[:id])        
  node[:id] = document.modified_id_to_paste(node[:id])
end
```